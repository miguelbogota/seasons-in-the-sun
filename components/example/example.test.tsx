import { render } from '@testing-library/react';

import { Example } from './example';

describe('Example', () => {
  it('should match snapshot', () => {
    const { container } = render(<Example />);
    expect(container).toMatchSnapshot();
  });
});
