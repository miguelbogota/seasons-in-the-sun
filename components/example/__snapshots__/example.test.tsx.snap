// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Example > should match snapshot 1`] = `
<div>
  <div>
    Example
  </div>
</div>
`;
